packages/cli/src/commands/init.ts

```diff
@@ -160,17 +164,33 @@ export default class Init extends Command {
       .then(async ({templateDisplay}) => {
         const templateName = (templateDisplay as string).split(' ')[0];
         if (templateName === 'Other') {
-          await this.observeTemplates([], flags);
+          await this.observeTemplates([]);
         } else {
           selectedTemplate = templates.find(({name}) => name === templateName);
-          await this.observeTemplates([selectedTemplate], flags);
+          await this.observeTemplates([selectedTemplate]);
         }
       });
     this.projectPath = await cloneProjectTemplate(this.location, this.project.name, selectedTemplate);
     await this.setupProject(flags);
+
+    // This should be all evm chains ?
```

Comment:
I think this is fine for now. If we want to do other chains we need datasource processors